4coder_vim.cpp

```diff
@@ -2103,6 +2201,11 @@ void vim_get_bindings(Bind_Helper* context) {
     bind(context, key_back, MDFR_NONE, backspace_char);
     bind(context, 'n', MDFR_CTRL, word_complete);
 
+    bind(context, 'j', MDFR_NONE, vim_start_exit_insert_chord);
+    bind(context, 'k', MDFR_NONE, vim_end_exit_insert_chord);
```

Comment:
I don't think this is a standard vim thing, so it probably doesn't belong in the 4vim base code. It should probably just be a customization on top of 4vim

Reply:
Yes you are totally right. I think that was some plugin i just got used to.